src/testRunner/unittests/tsserver/rename.ts

```diff
@@ -14,16 +14,7 @@ namespace ts.projectSystem {
                     canRename: false,
                     localizedErrorMessage: "You cannot rename this element."
                 },
-                locs: [{
-                    file: bTs.path,
-                    locs: [
-                        protocolRenameSpanFromSubstring({
-                            fileText: bTs.content,
-                            text: "./a",
-                            contextText: bTs.content
-                        })
-                    ]
-                }],

```

Comment:
i'm not sure but i think this part was just trying to match the result of `findRenameLocations` indirectly. doesn't look like _need_ this info.